fetch.js

```diff
@@ -177,6 +177,8 @@
         this._bodyText = body.toString()
       } else if (!body) {
         this._bodyText = ''
+      } else if (body.getReader) {
+        this.body = body
```

Comment:
Are we now allowing people to interface with `response.body` directly? Is this in the spec?